src/lib.rs

```diff
@@ -234,6 +237,10 @@ impl<'a> Into<webdriver::command::LocatorParameters> for Locator<'a> {
                 using: webdriver::common::LocatorStrategy::CSSSelector,
                 value: s.to_string(),
             },
+            Locator::Id(s) => webdriver::command::LocatorParameters {
+                using: webdriver::common::LocatorStrategy::XPath,
+                value: format!("//*[@id=\"{}\"]", s),
```

Comment:
I wonder if we need to do some escaping here.. I think it's probably fine for now, but with more advanced identifiers that could totally be the case.